config/locales/datetime/fr.yml

```diff
@@ -0,0 +1,11 @@
+---
+fr:
+  datetime:
+    dotiw:
+      minutes:
+        one: 1 minute
+        other: '%{count} minutes'
```

Comment:
For the failing `i18n_spec`, you'll need to add these keys in here:

https://github.com/18F/identity-idp/blob/b4f0a189c07305dfe2b6e2243b215c540f19f7b3/spec/i18n_spec.rb#L9-L21

The spec is trying to check that we don't have English text in non-English locale files, but it's not smart enough to know that some words are the same as their English counterparts in other languages.